plugins/android/src/main/java/androidProjectConfiguration.kt

```diff
@@ -51,10 +100,21 @@ fun Project.androidProjectConfiguration(
  */
 object Forma {
 
-    private lateinit var _configuration: FormaConfiguration
-    val configuration: FormaConfiguration get() = _configuration
+    private val configurations = mutableMapOf<Any, FormaConfiguration>()
 
-    fun store(configuration: FormaConfiguration) {
-        _configuration = configuration
+    var defaultConfiguration: FormaConfiguration
+        get() = configurations[DefaultConfigurationKey]
+            ?: error("Trying to access to any project configuration, but there is no one configuration")
```

Comment:
"Trying to access to any project configuration, but there is no one configuration" ->
"Default configuration is not set, <doc/example link>"